corehq/apps/data_dictionary/util.py

```diff
@@ -264,6 +266,14 @@ def save_case_property(name, case_type, domain=None, data_type=None,
         return gettext('Unable to save valid values longer than {} characters').format(max_len)
 
 
+def delete_case_property(name, case_type, domain):
+    try:
+        prop = CaseProperty.objects.get(name=name, case_type__name=case_type, case_type__domain=domain)
+    except CaseProperty.DoesNotExist:
+        return gettext('Case property does not exist and might have already been deleted.')
```

Comment:
I have not seen "might have already been deleted" as a part of an error message before.
I believe just "Case property does not exist" should suffice though if we still need that then should this be?
`Case property does not exist or might have already been deleted.`

Reply:
The reasoning for adding this was to avoid potential confusion for when two users decide to delete the same case property on the Data Dictionary. 

Thinking through it again however, simply having "Case property does not exit" does sound sufficient as it describes the situation of it already being deleted as well. I'll update this as per your suggestion.